### PR TITLE
[OBSDEF-6884] - add in missing imagePullSecrets settings for darksite installs

### PR DESCRIPTION
**Pull request by eddiepavkovic**
## Purpose
[OBSDEF-6884](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6884)
- adding in missing imagePullSecrets settings for darksite installs

## PR checklist
- [x] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
tbd...

### CODE REVIEW

File: bookkeeper-operator/Chart.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+appVersion: 0.1.4
 description: Bookkeeper Operator Helm chart for Kubernetes
 home: https://github.com/pravega/bookkeeper-operator
 icon: https://bookkeeper.apache.org/images/bookkeeper_logo_blk.png
 keywords:
 - log storage
 - stream storage
 name: bookkeeper-operator
-version: 0.1.3
+version: 0.1.4
```
Comment (realAaronWu) [inline bookkeeper-operator/Chart.yaml:11]: Can we keep the version as 0.1.3 so as to match with the actual operator image that we are using?

**Review by Bjohnson131 — APPROVED** [review]

File: bookkeeper-operator/Chart.yaml (hunk 1 above)
Comment (vgupta-mickey) [reply bookkeeper-operator/Chart.yaml:11]: If you change the chats, you need to bump it up

File: dcm/Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 apiVersion: v2
-appVersion: 0.70.0
+appVersion: 0.70.2
```
Comment (vgupta-mickey) [inline dcm/Chart.yaml:3]: Why 70.0 to 70.2?
Is this because of the dummy chart version 70.1?

**Review by vgupta-mickey — APPROVED** [review]

**Review by NguyenMartin — APPROVED** [review]

**Review by sumithreddy — APPROVED** [review]

File: dcm/Chart.yaml (hunk 2 above)
Comment (eddiepavkovic) [reply dcm/Chart.yaml:3]: yes

**Review by ben-schumacher — APPROVED** [review]

**Review by vgupta-mickey — APPROVED** [review]

**Review by vgupta-mickey — COMMENTED** [review]
looks good

**Pull request merged**